app/Main.hs

```diff
@@ -0,0 +1,17 @@
+module Main where
+
+import Control.Monad((>=>),forM_)
+import System.Environment(getArgs)
+
+import Data.Aeson
+import Data.OpenAPI.V3_0_0
+import Data.Yaml
+
+readOpenAPI :: String -> IO OpenAPIObject
+readOpenAPI t =  either (error . show) id
+             <$> (decodeFileEither t)
+
+main :: IO ()
+main = do
+  inputFiles <- getArgs
```

Comment:
In the absence of any input files, could you add a brief help message here? No need to make a full-blown cli, I think that'll do. Also, would it make sense to put a separator between the specs when printed (perhaps the file name)?